.gitignore

```diff
@@ -17,6 +17,8 @@ export/
 /tmp
 /runtime/sql/deps
 /runtime/release
+cli/pkg/
+rill
```

Comment:
This will exclude any folder called `rill`, even nested. Make it `/rill` instead